colorfield/validators.py

```diff
@@ -17,3 +17,50 @@
     _("Enter a valid hexa color, eg. #00000000"),
     "invalid",
 )
+
+COLOR_RGB_RE = re.compile(
+    # prefix and opening parenthesis
+    r"^rgb\("
+    # first number
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # second number
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # third number
+    r"(\d{1,3})"
+    # closing parenthesis
+    r"\)$"
+)
+color_rgb_validator = RegexValidator(
+    COLOR_RGB_RE,
+    _("Enter a valid rgb color, eg. rgb(128, 128, 128)"),
+    "invalid",
+)
+COLOR_RGBA_RE = re.compile(
+    # prefix and opening parenthesis
+    r"^rgba\("
+    # first number
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # second number
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # third number§
+    r"(\d{1,3})"
+    # comma and optional space
+    r",\s?"
+    # alpha channel: decimal number between 0 and 1
+    r"([0-1](\.\d+)?)"
```

Comment:
I would limit the decimals to 2 digits only here, do you agree?

Reply:
Yes. Sounds good. 

Also, this will accept decimal values > 1, like 1.5, which are technically invalid values. It might make the regex a bit too complicated to make it right, though.

Reply:
True, I don't think it's too complicated considering that only the zero can be followed by decimals.

Reply:
Up to you, it's not that important.

Reply:
Yea, just done it now. It was simpler than I expected.

Reply:
Just a last retouch to the regex and we can merge this.